docs/fides/docs/guides/utility_scripts.md

```diff
@@ -0,0 +1,51 @@
+# Configuring Fides via utility functions
+
+Fides is a complex product, and accurate configuration can be hard. Rather than using the built-in Fides postman collection, you may find it faster to build configuration scripts in Python, using the primitives provided in the `scripts/setup/` directory.
+
+## Available functions
+
+The available functions are listed below:
+
+| Function |
+|----------|
+| `setup.authentication.get_auth_header` |
+| `setup.healthcheck.check_health` |
+| `setup.s3_storage.create_s3_storage` |
+| `setup.policy.create_policy` |
+| `setup.rule.create_rule` |
+| `setup.rule_target.create_rule_target` |
+| `setup.email.create_email_integration` |
+| `setup.rule.create_rule_target` |
+| `setup.postgres_connector.create_postgres_connector` |
+| `setup.mongodb_connector.create_mongodb_connector` |
+| `setup.privacy_request.create_privacy_request` |
+| `setup.subject_identity_verification.verify_subject_identity` |
+
+Each script uses the Fides API to create its corresponding primitive object in Fides config.
+
+## Secrets management
+
+Some third party integrations require secrets to grant access. Custom scripts are configured to read these secrets as constant variables from a `scripts/setup/secrets.py` file. Due to its sensitive nature, this file is not supplied in the repository, and should instead be stored somewhere more secure such as a password manager. It must be a valid Python file and the format of the secrets should be:
```

Comment:
I tried to put a breadcrumb here so that our internal devs would know where the file was likely to be located internally. Should we state it more directly somewhere?

Reply:
Thanks, I think we don't need to now since the secrets can also be supplied as environment variables